docs/source/workflows/data_sources.rst

```diff
@@ -15,6 +15,10 @@ Uploading a new file with the same name will produce a new version of the file w
 The columns in the CSV are extracted and parsed by a mapping of column header names to user-created descriptors.
 Columns in the CSV that are not mapped with a descriptor are ignored.
 
+The CSV must be UTF-8 encoded ASCII text with CRLF line terminators.
+The best way to ensure this is to save the file in a "Comma Separated Values" format and not any of the other small CSV variations.
```

Comment:
I compared the working and failed files some more and the difference doesn't actually seem to be the encoding. They're both UTF-8 encoded according to Python
```
with open("file.csv") as f:
    print(f)
```
But when I use the terminal command `file file.csv`, the failed files just say "ASCII text" while the working files say "ASCII text, with CRLF line terminators"

@andyczerwonka @talltea do you have a concise explanation for what this difference?